schema-loader/build.gradle

```diff
@@ -153,90 +137,59 @@ shadowJar {
     mergeServiceFiles()
 }
 
-// The fat jar path
-def schemaLoaderJarPath = "${project.buildDir}/libs/${archivesBaseName}-${project.version}.jar"
-
 task integrationTestCassandra(type: Test) {
-    dependsOn shadowJar
```

Comment:
Since the integration tests no longer depend on the fat jar after this change, I removed this dependency.